### ISSUE TITLE
Melee bots cannot reach target properly

### ISSUE THREAD

**Issue by ike3** [ike3/mangosbot#9]
Although AiPlayerbot.MeleeDistance = 2.0, bots some times stay just outside of melee distance and cannot attack the target. Seems something prevents them to reach target properly and start swinging their weapons.

**Comment by ike3** [comment]
Seems this almost fixes the issue but I don't think the distance is calculated correctly

```
AiPlayerbot.MeleeDistance = 0.5
AiPlayerbot.FollowDistance = 0.5
```

**Comment by dmanbob** [comment]
OK thanks I'm going to try this a s well. I noticed too they were not getting close sometimes.

**Comment by ike3** [comment]
Not a best fix but almost never reproduces after [ai-0.36-64]
